index.js

```diff
@@ -0,0 +1,91 @@
+import express from 'express';
+import fs from 'fs';
+
+const server = express();
+
+let requestExpirationDates = [];
+let nextExpirationDate = new Date();
+let nextComparsionInterval = 1;
+
+setInterval(() => { 
+    if(nextExpirationDate <= new Date() && requestExpirationDates.length !== 0){
+      requestExpirationDates.every(element => {
+        if(element <= nextExpirationDate){
+          requestExpirationDates.shift();
```

Comment:
I like the concept of cleaning up the file to keep it from growing endlessly :+1: